docusaurus/docs/quickstarts/network/hosted/index.md

```diff
@@ -1,168 +1,258 @@
-After the files are generated, you can then install them for use by systemd by running:
-
-```bash
-sudo cp "${ZITI_HOME}/${ZITI_EDGE_CONTROLLER_RAWNAME}.service" /etc/systemd/system/ziti-controller.service
-sudo cp "${ZITI_HOME}/${ZITI_EDGE_ROUTER_RAWNAME}.service" /etc/systemd/system/ziti-router.service
-sudo systemctl daemon-reload
-sudo systemctl start ziti-controller
-sudo systemctl start ziti-router
-```
-
-Now, both the controller and the edge router will restart automatically!  After a few seconds you can then run these
-commands and verify systemd has started the processes and see the status:
-
-```bash
-sudo systemctl -q status ziti-controller --lines=0 --no-pager
-sudo systemctl -q status ziti-router --lines=0 --no-pager
-
-● ziti-controller.service - Ziti-Controller
-     Loaded: loaded (/etc/systemd/system/ziti-controller.service; disabled; vendor preset: enabled)
-     Active: active (running) since Thu 2021-11-11 19:05:46 UTC; 8s ago
-   Main PID: 2375 (ziti-controller)
-      Tasks: 7 (limit: 1154)
-     Memory: 43.7M
-     CGroup: /system.slice/ziti-controller.service
-             └─2375 /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.22.11/ziti-controller run /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/co…
-ubuntu@ip-10-0-0-1:~$ sudo systemctl -q status ziti-router --lines=0 --no-pager
-● ziti-router.service - Ziti-Router for ip-10-0-0-1-edge-router
-     Loaded: loaded (/etc/systemd/system/ziti-router.service; disabled; vendor preset: enabled)
-     Active: active (running) since Thu 2021-11-11 19:05:47 UTC; 8s ago
-   Main PID: 2385 (ziti-router)
-      Tasks: 6 (limit: 1154)
-     Memory: 231.4M
-     CGroup: /system.slice/ziti-router.service
-             └─2385 /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.22.11/ziti-router run /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ip-10…
-```
-
-## Adding Environment Variables Back to the Shell
-
-If you log out and log back in again you can source the '.env' file located at:
-`. ~/.ziti/quickstart/$(hostname)/$(hostname).env`. Below is an example of logging in, not having ZITI_HOME set, sourcing the
-.env file and then showing ZITI_HOME set:
-
-```bash
-Last login: Thu May 20 11:36:25 2021 from 67.246.244.61
-ubuntu@ip-10-0-0-1:~$ echo $ZITI_HOME
-
-ubuntu@ip-10-0-0-1:~$ . ~/.ziti/quickstart/$(hostname)/$(hostname).env
-
-adding /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.20.2 to the path
-ubuntu@ip-10-0-0-1:~$ echo $ZITI_HOME
-/home/ubuntu/.ziti/quickstart/ip-10-0-0-1
-```
-
-## Install Ziti Admin Console (ZAC) [Optional]
-
-Once you have the network up and running, if you want to install the UI management console, the ZAC, [follow along with
-the installation guide](../zac/installation)
-
-## Using the Overlay
-
-Now you have your zero trust overlay network in place, you probably want to try it out. Head on over to
-[the services quickstart](../services) and start the journey to understanding how to use OpenZiti.
+---
+sidebar_position: 10
+---
```

Comment:
Because you'd suggested separate pages for the next steps, but there's not quite enough content yet to break those out into pages. I'd already moved hosted.md to hosted/index.md by the time I realized that, and expect it'll need to be moved anyway to facilitate created those forthcoming child pages for next steps.